open_prices/challenges/models.py

```diff
@@ -0,0 +1,81 @@
+from django.contrib.postgres.fields import ArrayField
+from django.core.validators import ValidationError
+from django.db import models
+from django.utils import timezone
+
+from open_prices.challenges import constants as challenge_constants
+from open_prices.common import utils
+
+
+class ChallengeQuerySet(models.QuerySet):
+    def published(self):
+        return self.filter(is_published=True)
+
+
+class Challenge(models.Model):
```

Comment:
did a couple of changes
- removed the `challenge_id` field (there is already an auto-generated `id` field)
- removed the `state` field and added a `status` property
- added a `is_published` boolean field
- added some validation on save (start & end dates, some fields required if published)